experimental/BasicAsyncHttpServer/BasicAsyncHttpServer.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFramework>net451</TargetFramework>
+    <!-- TODO remove rid when https://github.com/dotnet/sdk/issues/396 is resolved -->
+    <RuntimeIdentifier Condition="'$(TargetFramework)'!='netcoreapp1.1'">win7-x64</RuntimeIdentifier>
+    <WarningsAsErrors>true</WarningsAsErrors>
```

Comment:
(affects all csproj files in PR). Should be TreatWarningsAsErrors. Also, you shouldn't need it. By referencing Internal.AspNetCore.Sdk, this is on by default.